agent/engine/docker_task_engine.go

```diff
@@ -553,6 +577,20 @@ func (engine *DockerTaskEngine) pullAndUpdateContainerReference(task *api.Task,
 		return DockerContainerMetadata{Error: TaskStoppedBeforePullBeginError{task.Arn}}
 	}
 
+	// Set the credentials for pull from ecr if necessary
+	if container.ShouldPullWithExecutionRole() {
+		credential, ok := engine.credentialsManager.GetTaskCredentials(task.GetExecutionCredentialsID())
+		if !ok {
+			seelog.Infof("Acquiring ecr credentials from credential manager failed, container: %s, task: %s", container.String(), task.String())
+			return DockerContainerMetadata{Error: CannotPullECRContainerError{errors.New("Acquiring ecr credentials failed, credentials not found in credential manager")}}
```

Comment:
error message starts with upper case letter. Also, you can rephrase that as `"engine ecr credentials acquisition: not found in credentials manager"`